rust-symcrypt/BUILD.md

```diff
@@ -6,17 +6,19 @@ who want to use their own `symcrypt*` files.
 ## Installation
 To use the SymCrypt crate, you must have a local version of [SymCrypt](https://github.com/microsoft/SymCrypt) downloaded.
 
-Please follow the [Build Instructions](https://github.com/microsoft/SymCrypt/blob/main/BUILD.md) that is provided by SymCrypt to install SymCrypt for your target architecture.
+For ease of use, the recommended usage is to obtain these binaries from the official SymCrypt [Repo](https://github.com/microsoft/SymCrypt/releases/tag/v103.4.2).
 
-Once SymCrypt is installed and built locally on your machine, we must configure your machine so that the SymCrypt crate's build script can easily find `symcrypttestmodule.dll` and `symcrypttestmodule.lib` which are needed on Windows, or the `libsymcrypt.so*` files which are needed for Linux. 
+If you wish to build your own version please follow the [Build Instructions](https://github.com/microsoft/SymCrypt/blob/main/BUILD.md) that is provided by SymCrypt to install SymCrypt for your target architecture.
+
+Once SymCrypt is installed and built locally on your machine, we must configure your machine so that the SymCrypt crate's build script can easily find `symcrypt.dll` and `symcrypt.lib` which are needed on Windows, or the `libsymcrypt.so*` files which are needed for Linux. 
 
 ### Windows Install 
 
-The `symcrypttestmodule.lib` can be found in the the following path after SymCrypt has been successfully downloaded and built. 
+The `symcrypt.lib` can be found in the the following path after SymCrypt has been successfully downloaded and built. 
 
 `C:\Your-Path-To-SymCrypt\SymCrypt\bin\lib`
 
-The SymCrypt crate needs a static lib to link to during its build/link time. You must configure your system so that the SymCrypt crate's build script can easily find the needed `symcrypttestmodule.lib` file.
+The SymCrypt crate needs a static lib to link to during its build/link time. You must configure your system so that the SymCrypt crate's build script can easily find the needed `symcrypt.lib` file.

```

Comment:
This is the wrong path. If you build SymCrypt locally, there will be two distinct `symcrypt.lib` files:

1. `lib\symcrypt.lib` - this is the full static library which you would use for static linking
2. `dll\symcrypt.lib` - this is the DLL import library which is needed to dynamically link the DLL

We want to dynamically link to symcrypt.dll, so the second one is the one you want. Although, it's also possible to generate an import library directly from the DLL, and I thought bindgen supported that, so you might not really need the import library at all?

Reply:
Im not sure about bindgen being able to generate the import library directly from the DLL, but I've just tried removing the `symcrypt.lib` from the directory and my tests fail, with error: `note: LINK : fatal error LNK1181: cannot open input file 'symcrypt.lib'`

So I think that the lib is needed. 

As for the paths I will update it to be `dll\symcrypt.lib`

I will also look into bindgen generating its own lib from the `symcrypt.dll` provided.

Reply:
Path looks good, but I'd change the first sentence here to:

"The SymCrypt crate needs to link against the SymCrypt import library during build."

Reply:
Updated the line to match your suggestion